integration/kubernetes/helm-chart/alluxio/CHANGELOG.md

```diff
@@ -258,3 +258,7 @@
 0.6.40
 
 - Fix incorrect directory when mounting & formatting master journal volume
+
+0.6.41
+
+- Support alluxio proxy.
```

Comment:
pls double check if 0.6.41 is still the latest version

Reply:
Bump